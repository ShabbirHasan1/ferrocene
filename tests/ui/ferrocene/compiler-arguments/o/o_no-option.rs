// This test should check what happen when no option is provided to -o
// Unfortunately, the test frameworking is adding other options after it,
// causing an unexpected failure.
//
// check-fail
// compile-flags: -o

fn main() {}

// ferrocene-annotations: um_rustc_o
