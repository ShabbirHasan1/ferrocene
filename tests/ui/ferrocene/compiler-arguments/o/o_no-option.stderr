error: Argument to option 'o' missing

